scanner/src/main/kotlin/LocalScanner.kt

```diff
@@ -56,6 +56,7 @@ import org.ossreviewtoolkit.model.ScannerRun
 import org.ossreviewtoolkit.model.Severity
 import org.ossreviewtoolkit.model.Success
 import org.ossreviewtoolkit.model.VcsInfo
+import org.ossreviewtoolkit.model.VcsType
```

Comment:
Commit message: Replace "is" with "it" in first line.

Reply:
Done.